src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const toDo of actions) {
+    switch (toDo.type) {
+      case 'addProperties':
+        Object.assign(state, toDo.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const remove of toDo.keysToRemove) {
+          delete state[remove];
+        }
+        break;
+
+      case 'clear':
+        for (const clear in state) {
+          delete state[clear];
+        }
+
+        break;
```

Comment:
Рекомендую использовать фигурные скобки в case, так будет проще и легче понять, к чему относится этот break
```suggestion
      case 'clear': {
        for (const clear in state) {
          delete state[clear];
        }

        break;
        }
```

Reply:
that's not necessarily